service/controller/chart/resource/status/create.go

```diff
@@ -83,6 +83,7 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 		Reason:     reason,
 		Release: v1alpha1.ChartStatusRelease{
 			LastDeployed: metav1.Time{Time: releaseContent.LastDeployed},
+			Revision:     releaseContent.Revision,
```

Comment:
I remember what happened now. With Helm 2 we needed to call both `GetReleaseContent` and `GetReleaseHistory`. 

With Helm 3 we only need to call `GetReleaseContent` but I missed setting this field.